taxcalc/params.json

```diff
@@ -394,9 +394,9 @@
     },
      "_EITC_rt":{
         "long_name": "Earned income credit rate",
-	    "description": "Apply this rate on Income.",
+	"description": "If one taxpayer's AGI is below AGI phaseout start, he or she can apply this rate to your earned income to calculate the credit amount eligible. ",
```

Comment:
On Tue, 4 Aug 2015, Matt Jensen wrote:

> 
> In taxcalc/params.json:
> 
> > @@ -394,9 +394,9 @@
> >      },
> >       "_EITC_rt":{
> >          "long_name": "Earned income credit rate",
> > -	    "description": "Apply this rate on Income.",
> > +	"description": "If one taxpayer's AGI is below AGI phaseout start, he or 
> she can apply this rate to your earned income to calculate the credit amount
>  eligible. ",
> 
> Maybe -> "If one taxpayer's AGI is below AGI phaseout start, he can apply
> this rate to his earned income to calculate the credit amount eligible. "

There is no separate AGI calculation for husband and wife. I would say 
"Product of this parameter and  earned income is the Earned INcome Tax 
Credit before the phaseout based on AGI."
> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVQmBXSfud5Hueey8mfeNr5oPmvJcks5okKEqgaJpZM4FhcK0.gif]
> 
> 
>